nav2_bt_navigator/README.md

```diff
@@ -154,11 +139,11 @@ Image below depicts the graphical version of the complete Navigation Task with A
 
 <img src="./doc/AutoLocalization_w_recovery_parallel.png" title="Navigation Behavior Tree with AutoLocalization, Recovery, and Planning & Control" width="70%" align="middle">
 
+## Future Work
+Currently, in our navigation stack global recoverable actions are implemented.  However, a better approach is to implement both local and global recovery actions.  The local recovery actions would be responsible for recovering each module locally and if that fails the failure should propagate up to the tree to invoke global recovery actions. Global recovery actions could be recovery actions such as re-initializing system, re-calibrating robot, bringing the system to a good known state, etc.  A simple global and local recoverable node is depicted in the diagram below.
```

Comment:
Might want to explain the "local and global recoverable actions" a bit more. Readers could confuse this as being related to global and local planning. Perhaps explain the idea that error handling can occur at multiple scopes.